docs/ops.md

```diff
@@ -0,0 +1,33 @@
+# Operational Docs
+
+## Cleaning up a failed migration
+
+Use these instructions if a migration fails, and you want to retry it
+(e.g. you found and fixed the issue that caused it to fail).
+*note: if the migrator called `update` on the migration instance, this could get much more complicated, and fixing forward might be better*
+
+1. Get the guid of the migration service instance:
+   ```
+   $ cf service <instance-name> --guid
+   ```
+2. Purge the new service instance:
+   ```
+   $ cf purge-service-instance <migration instance name>
+   ```
+3. If the migrator called `update` on the migration instance, you'll need to 
```

Comment:
Is step 3 just referring to moving along to steps 4 and 5, or is there a part missing?

Reply:
step 3 is what happens when you switch tasks mid-sentence 👎